finetrainers/args.py

```diff
@@ -391,6 +392,41 @@ def _add_diffusion_arguments(parser: argparse.ArgumentParser) -> None:
     )
 
 
+def _add_regularization_arguments(parser: argparse.ArgumentParser) -> None:
+    parser.add_argument(
+        "--caption_dropout_p",
+        type=float,
+        default=0.00,
+        help="Probability of dropout for the caption tokens.",
+    )
+    parser.add_argument(
+        "--caption_dropout_technique",
+        type=str,
+        default="empty",
+        choices=["empty", "zero"],
+        help="Technique to use for caption dropout.",
+    )
+    parser.add_argument(
+        "--image_condition_dropout_p",
+        type=float,
+        default=0.00,
+        help="Probability of dropout for the image condition latents. This will replace the image latent condition with a zero tensor.",
+    )
+    parser.add_argument(
+        "--image_condition_noise_scale",
+        type=float,
+        default=0.00,
+        help="Maximum scale of noise to add to the image condition latents. A random noise tensor will be added to the image condition latents at at strength between 0 and this value.",
+    )
+    parser.add_argument(
+        "--image_condition_noise_type",
+        type=str,
+        default="gaussian",
+        choices=["gaussian"],
+        help="Type of noise to add to the image condition latents. Defaults to plain Gaussian noise.",
+    )
```

Comment:
Does this need to be different for Cog I2V?

Reply:
I added this for testing purposes. I don't think I've been able to see its effectiveness with LTX yet so I will actually remove it.

Cog uses pre-encoding noise, while here I'm using post-encoding noise addition. This is inspired by what the LTX folks do in the original repository for finetuning